static/js/components/NewShift.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Form from "@rjsf/material-ui";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import { showNotification } from "baselayer/components/Notifications";
+import { submitShift } from "../ducks/shift";
+import { fetchShifts } from "../ducks/shifts";
+
+const NewShift = () => {
+  const groups = useSelector((state) => state.groups.userAccessible);
+
+  const dispatch = useDispatch();
+
+  if (!groups) {
+    return <CircularProgress />;
+  }
```

Comment:
I cannot submit the form because the date picker and tabbing wont let me fill out the seconds (in Firefox). Perhaps dont require/allow seconds?
![image](https://user-images.githubusercontent.com/480799/152059403-71b321af-3667-4360-8c03-0cca63992494.png)

Reply:
Its actually AM and PM here.